### PR TITLE
Declare Bio.Application obsolete

### PR DESCRIPTION
**Pull request by peterjc**
See discussion on #2877.

- [x] I hereby agree to dual licence this and any previous contributions under both
the _Biopython License Agreement_ **AND** the _BSD 3-Clause License_.

- [x] I have read the ``CONTRIBUTING.rst`` file, have run ``pre-commit`` locally,
and understand that AppVeyor and TravisCI will be used to confirm the Biopython unit
tests and style checks pass with these changes.

- [x] I have added my name to the alphabetical contributors listings in the files
``NEWS.rst`` and ``CONTRIB.rst`` as part of this pull request, am listed
already, or do not wish to be listed. (*This acknowledgement is optional.*)

### CODE REVIEW

File: Doc/Tutorial/chapter_align.tex [hunk 1]
```diff
@@ -928,6 +928,10 @@ \section{Alignment Tools}
 output filename), then invoke this command line via a Python operating system call (e.g.
 using the \texttt{subprocess} module).
 
+\emph{WARNING:} We have decided to drop these command line wrappers in a future Biopython
+release. We will be updating this documentation to instead build the command line
```
Comment (mdehoon) [inline Doc/Tutorial/chapter_align.tex:932]: Don't promise too much..

**Review by mdehoon — APPROVED** [review]

File: Doc/Tutorial/chapter_align.tex (hunk 1 above)
Comment (peterjc) [reply Doc/Tutorial/chapter_align.tex:932]: Well, it might turn out to be some examples updated and many examples removed... but I'd hope not.

**Comment by peterjc** [comment]
Anyone want to give a second opinion? @chapmanb maybe?

**Comment by peterjc** [comment]
It wasn't to the mailing list, but Brad seems OK with the idea in principle, saying it "doesn't make sense to support a Biopython specific OO framework for doing this".

I'll rebase this now to sort out the flake8 warning on CircleCI

**Comment by peterjc** [comment]
Don't see why CircleCI says it can't see the configuration... that is working on the master branch

**Pull request merged**